### ISSUE TITLE
Starred playlist raises error

### ISSUE THREAD

**Issue by frafra** [jrnewell/spotify-ripper#59]
```
Loading starred playlist...
Exception in thread Thread-1:
Traceback (most recent call last):
  File "/usr/lib64/python2.7/threading.py", line 813, in __bootstrap_inner
    self.run()
  File "/usr/lib/python2.7/site-packages/spotify_ripper/ripper.py", line 131, in run
    tracks = self.load_link(uri)
  File "/usr/lib/python2.7/site-packages/spotify_ripper/ripper.py", line 208, in load_link
    starred.load()
AttributeError: 'NoneType' object has no attribute 'load'
```

**Comment by jrnewell** [comment]
Fixed in version ``2.1.2``

**Comment by MasterSmollett** [comment]
On version 2.7.0, not working...
